src/xbps/repositories/index.md

```diff
@@ -71,8 +71,9 @@ Once enabled, symbols may be obtained for `<package>` by installing
 
 #### Finding debug dependencies
 
-The `xtools` package contains the `xdbg` utility to retrieve a list of debug
-packages, including dependencies, for a package:
+The `xtools` package contains the [xdbg](https://man.voidlinux.org/xtools.1)
```

Comment:
`xdbg(1)`, right?

Reply:
i thought about that. Since the link is taking one to the `xtools.1` man page, and `xdbg.1` doesn't actually exist, i chose to omit the "(1)". i have no idea what the correct thing to do is in this case.

Reply:
That's a good point... I would prefer having the `(1)` for consistency, and it works.